Cargo.toml

```diff
@@ -27,3 +27,4 @@ cplex = ["xml-rs"]
 [dependencies]
 tempfile = "3"
 xml-rs = { version = "0.8.3", optional = true }
+quick-xml = "0.31"
```

Comment:
Do we need **both** xml-rs and quick-xml ? If the latter is better, then we should probably just use it everywhere. Depending on two different xml parsing libraries isn't great...

Reply:
Yep good point. Lemme see it that can be removed directly or I need to change something else

Reply:
Looks like it was not used anywhere else, removed it in the latest commit